rand_core/src/lib.rs

```diff
@@ -306,6 +306,10 @@ impl<R: TryCryptoRng> CryptoRng for UnwrapErr<R> {}
 /// This trait encapsulates the low-level functionality common to all
 /// pseudo-random number generators (PRNGs, or algorithmic generators).
 ///
+/// Note that this trait does not imply
+/// [reproducibility](https://rust-random.github.io/book/crate-reprod.html)
+/// of results.
```

Comment:
Maybe +", though individual types may provide their own guarantees for their implementation of this trait."? Since there are certainly rngs that do guarantee reproducibility with the same given seed.

Reply:
Correct; it's really just `SmallRng` and `StdRng` (of our RNGs) that don't provide reproducibility.

Reply:
I'm removing this from the PR; see #1570 instead. (If it doesn't get added, something similar to the first commit in that PR can be.)